src/server_manager/web_app/app.spec.ts

```diff
@@ -35,41 +32,43 @@ const TOKEN_WITH_ONE_SERVER = 'one-server-token';
 // tslint:disable-next-line:no-any
 (global as any).bringToFront = () => {};
 
+// Inject app-root element into DOM once before the test suite runs
```

Comment:
I wonder if theres a better way to load the app-root element. Maybe we should even try load the whole index.html.